docs/examples/load-query-and-chat-marqo.md

```diff
@@ -18,13 +17,10 @@ vector_store = MarqoVectorStoreDriver(
     index=os.environ["MARQO_INDEX_NAME"],
     embedding_driver=OpenAiEmbeddingDriver()
 )
-# Initialize the query engine
-query_engine = VectorQueryEngine(vector_store_driver=vector_store, prompt_driver=OpenAiChatPromptDriver(model="gpt-3.5-turbo"))
 
 # Initialize the knowledge base tool
 vector_store_tool = VectorStoreClient(
     description="Contains information about the Griptape Framework from www.griptape.ai",
-    query_engine=query_engine,
```

Comment:
Don't we need to pass in a Vector Store Driver now?

Reply:
Yep! Will fix it in the followup docs PR.